client/src/test/java/com/sinch/sdk/domains/verification/adapters/VerificationsReportServiceTest.java

```diff
@@ -30,7 +30,7 @@
 @TestWithResources
 public class VerificationsReportServiceTest extends BaseTest {
 
-  @GivenJsonResource("/domains/verification/v1/report/VerificationReportCalloutRequestDto.json")
+  @GivenJsonResource("/domains/verification/v1/report/VerificationReportPhoneCallRequestDto.json")
   public VerificationReportRequestParametersInternal verificationReportCalloutRequestDto;
```

Comment:
You may also rename the corresponding variable.

Reply:
done